keyboards/hecomi/keymaps/kakunpc/keymap.c

```diff
@@ -64,29 +52,11 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
You can actually remove all of these functions if you're not doing anything with them :)